src/printer.js

```diff
@@ -1053,6 +1053,8 @@ function printNode(path, options, print) {
         return "array";
       } else if (node.name === "\\callable") {
         return "callable";
+      } else if (node.name.toLowerCase() === "null") {
```

Comment:
By PSR-2

Reply:
can you add a comment explaining we prefer lowercase?